app.py

```diff
@@ -824,6 +824,16 @@ def _remove_punct(tree):
 	_remove_punct(tree_copy)
 	return number_terminals(prune_empty_non_terminals(tree_copy))
 
+def ptb_to_ptree(ptb_tree: str):
```

Comment:
would you mind adding a docstring explaining that this is because the PTB conversion implemented in cgel.py (1) doesn't attach the `p` function to punctuation nodes and (2) doesn't implement the special punctuation tags `LRB`, `RRB`, and `HYPH`

Reply:
actually I'll do it